.circleci/config.yml

```diff
@@ -13,17 +13,78 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-          - v1-dependencies-{{ checksum "yarn.lock" }}
-          - v1-dependencies-
+            - v1-dependencies-{{ checksum "yarn.lock" }}
+            - v1-dependencies-
       - run: yarn install
       - save_cache:
           paths:
             - node_modules
           key: v1-dependencies-{{ checksum "yarn.lock" }}
       - run: yarn test-ci
 
+  build:graphql:
+    docker:
+      -image: circleci/node:10
+    working_directory: ~/margarita
+    steps:
+      - checkout
+      - retore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "yarn.lock" }}
+            - v1-dependencies-
+      - run: yarn install
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "yarn.lock" }}
+      - run:
+          name: Build GraphQL lambda
+          command: yarn build:graphql
+
+    deploy:graphql:
+      docker:
+        -image: circleci/node:10
+      working_directory: ~/margarita
+      steps:
+        - checkout
+        - retore_cache:
+            keys:
+              - v1-dependencies-{{ checksum "yarn.lock" }}
+              - v1-dependencies-
+        - run:
+          name: Install Serverless CLI and dependencies
+          command: |
+            sudo npm i -g serverless
```

Comment:
should we use `yarn global add` instead?

Reply:
I took what was written on the CircleCI docs/tutorial. Do you think it makes any difference?

Reply:
Probably not